jfr-daemon/build.gradle.kts

```diff
@@ -30,6 +30,9 @@ dependencies {
 }
 
 tasks.jar {
+    // Create a standard jar as well as a shadowJar, add a classifier to differentiate
+//    enabled = true
+//    archiveClassifier.set("regular")
 // Create shadowJar instead of jar
     enabled = false
```

Comment:
I believe you'll have to resurrect this classifier configuration. See this [convo](https://newrelic.slack.com/archives/CV2EXQQA2/p1615834318076000?thread_ts=1615825528.071200&cid=CV2EXQQA2) for details.